src/server/worker/common/common.go

```diff
@@ -37,9 +37,17 @@ func DatumID(inputs []*Input) string {
 		hash.Write([]byte(input.Name))
 		_ = binary.Write(hash, binary.BigEndian, int64(len(input.Name)))
 		file := input.FileInfo.File
-		hash.Write([]byte(file.Commit.Branch.Repo.Name))
-		_ = binary.Write(hash, binary.BigEndian, int64(len(file.Commit.Branch.Repo.Name)))
-		hash.Write([]byte(file.Commit.Branch.Name))
+		// TODO(acohen4): this isn't cute
```

Comment:
Undone TODO

Reply:
We should be able to make the necessary changes to just use the repo field internally.